Documentation/minio-object-store-crd.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Minio Object Store CRD
+weight: 40
+indent: true
+---
+
+# Minio Object Store CRD
+Minio object stores can be created and configured using the `objectstores.minio.rook.io` custom resource definition (CRD). Complete instructions can be found in the [Rook Minio Documentation](minio-object-store.md).
+
+##Sample
+
+```yaml
```

Comment:
this should be a sample of a CRD instance, not the CRD type.  so basically what's in `cluster/examples/kubernetes/minio/object-store.yaml`.